streampipes-client-python/streampipes/function_zoo/river_function.py

```diff
@@ -187,7 +188,12 @@ def __init__(
             attributes["truth"] = prediction_type
             if target_label is None:
                 raise ValueError("You must define a target attribute for a supervised model.")
-        output_stream = create_data_stream("prediction", attributes)
+
+        output_stream = create_data_stream(
+            name="prediction",
+            attributes=attributes,
+            broker=get_broker_enum(client.dataStreamApi.get(stream_ids[0])),  # type: ignore
```

Comment:
there shouldn't be a need to mute mypy here

Reply:
The problem is that `get` returns a `Ressource`, but `get_broker_enum` expects a `DataStream`